internal/node/reverse_proxy/proxy_test.go

```diff
@@ -40,81 +44,83 @@ func TestProxy(t *testing.T) {
 	file.Close()
 
 	// Create proxy server
-	proxy := &ProxyServer{
-		Rules: make(RuleSet),
-	}
+	proxy := NewProxyServer(logging.NewLogger(), &config.NodeConfig{})
 	err = proxy.Rules.Add("backend1", &RedirectRule{
 		Matcher:         "/backend1",
 		ForwardLocation: redirectedServerURL,
 	})
-	assert.Nil(t, err)
+	require.Nil(t, err)
```

Comment:
Why is this changing assert to require?

Reply:
https://www.yellowduck.be/posts/assert-vs-require-in-testify#:~:text=Which%20one%20you%20choose%20depends%20on%20the%20behaviour%20you%20are%20looking%20for.&text=With%20assert%20%2C%20you%20will%20get%20a%20log%20entry%20for%20each%20assertation.&text=With%20require%20%2C%20as%20soon%20as,the%20test%20will%20be%20skipped.

Oh nice I'm a fan, thanks for doing this